cypress_test/integration_tests/mobile/calc/hamburger_menu_spec.js

```diff
@@ -528,9 +528,9 @@ describe('Trigger hamburger menu options.', function() {
 
 		cy.get('@origHeaderHeight')
 			.then(function(origHeaderHeight) {
-				cy.get('.spreadsheet-header-columns')
+				cy.get('[id="test-div-column header"]')
 					.should(function(header) {
-						expect(header.height()).to.be.greaterThan(origHeaderHeight);
+						expect(header.height()).to.be.equal(origHeaderHeight);
```

Comment:
Here, it would be better to use something else than origHeaderHeight. Earlier, grouping was drawn inside the column header DOM element, so the header's height was a good indicator for the grouping visibility. However, checking equality here means, that the header's height does not change anymore, so this won't actually test that the grouping is there or not.
As I see with this PR, you introduced a new canvas section for grouping. So you should use that section to check whether enabling / disabling grouping has a result on the document. It might be enough to check the existence of the grouping section or the size of it, whatever indicates that the grouping is visible.
There are more test cases with the same issue: 'Data: grouping / ungrouping.', 'Data: remove grouping outline.', 'Data: show / hide grouping details.'.